packages/ui-tree-browser/src/TreeBrowser/index.js

```diff
@@ -358,6 +354,11 @@ class TreeBrowser extends Component {
     }
   }
 
+  getIsFlattened = (collection) => {
+    const { rootId, showRootCollection } = this.props
+    return !showRootCollection && rootId && collection.id === rootId
```

Comment:
I'd put here a warning if showRootCollection is true, but rootId is null/NaN

Reply:
That is a valid case too, actually the default case... Renders all collections at the top level.